src/main/java/com/anchore/jenkins/plugins/anchore/AnchoreBuilder.java

```diff
@@ -66,7 +66,7 @@ public class AnchoreBuilder extends Builder implements SimpleBuildStep {
   private boolean autoSubscribeTagUpdates = DescriptorImpl.DEFAULT_AUTOSUBSCRIBE_TAG_UPDATES;
   private boolean forceAnalyze = DescriptorImpl.DEFAULT_FORCE_ANALYZE;
 
-  // Override global config. Supported for anchore-engine mode config only
+  // Override global config. Supported for anchore-enterprise mode config only
   private String engineurl = DescriptorImpl.EMPTY_STRING;
```

Comment:
I know it has no functional gain but given we're already changing some of these is it work s/engineurl/enterpriseurl and all the others too? Totally fine if you want to leave out of this change though